OrbitQt/orbitmainwindow.ui

```diff
@@ -425,6 +425,7 @@ QPushButton:disabled {
     <property name="title">
      <string>Help</string>
     </property>
+    <addaction name="actionOpen_User_Data_Directory"/>
```

Comment:
Nit: I think we should get rid of those auto-generated names and go for a consistent style. To my understanding, the majority of our QT components are named in lower camel case (`actionServiceStackOverflow`).